src/docs/product/issues/performance-issues/n-one-queries.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "N+1 Queries"
+sidebar_order: 10
+description: "Learn more about N+1 queries and how to diagnose and fix them."
+---
+
+<Include name="early-adopter-note.mdx" />
+
+## N+1 Queries
+
+"N+1 Queries" is a performance problem where the application makes database queries in a loop, instead of making a single query that returns all the information at once. Each database connection has some amount of performance overhead, so querying the database in a loop can be many times slower than doing it just once. This problem often occurs when using an Object-relational mapping (ORM) tool in web frameworks like Django or Ruby on Rails.
+
+## Span Evidence
+
+The evidence for a N+1 queries has four main aspects:
+
+- the transaction name
+- the parent span. This can be a view, a serializer, or another span that logically groups the queries
+- the source span. This is the query that caused the N+1 queries
+- the repeating span. This is the "N" of "N+1 queries". This is the looped query that should have been part of the source
```

Comment:
```suggestion
- Transaction name
- Parent span -  This can be a view, a serializer, or another span that logically groups the queries.
- Source span - This is the query that caused the N+1 queries.
- Repeating span - This is the "N" of N+1 queries. This is the looped query that should have been part of the source.
```